packages/menu-bar/theme/material/vaadin-menu-bar-button-styles.js

```diff
@@ -73,7 +73,7 @@ const menuBarButton = css`
   }
 
   :host([theme~='outlined']:not([dir='rtl']):nth-last-of-type(2)),
-  :host([theme~='outlined']:not([dir='rtl'])[part~='overflow-button']) {
+  :host([theme~='outlined']:not([dir='rtl'])[slot='overflow']) {
```

Comment:
Is the overflow part needed in this selector in light of the fact that we already have `margin-inline-end: 0` [here](https://github.com/vaadin/web-components/pull/4687/files#diff-d176f273fde0dbcc6f664b9e8774c60277424fef228996b17f00c010a54f67aeR16-R18)?

Reply:
Yes, it's still needed because of this style:

https://github.com/vaadin/web-components/blob/016299b5390e8b6691b13b49e0e4ca91cfadb5ee/packages/menu-bar/theme/material/vaadin-menu-bar-button-styles.js#L71-L73